src/nupic/bindings/algorithms.i

```diff
@@ -636,91 +636,6 @@ void forceRetentionOfImageSensorLiteLibrary(void) {
 }
 
 //--------------------------------------------------------------------------------
-// LearningSet for continuous FDR TP
```

Comment:
The two Inhibition classes being extended were removed a long time ago so this SWIG code for extending them was throwing a warning. And the function wasn't used anywhere so I am removing it as well.